c/httpfileservice.c

```diff
@@ -200,13 +200,21 @@ void deleteUnixDirectoryAndRespond(HttpResponse *response, char *absolutePath) {
 void createFileFromUnixDirectoryAndRespond(HttpResponse *response, char *absolutePath) {
   if (isDir(absolutePath)) {
       int folderNameLen = strlen(absolutePath);
+      if(folderNameLen == 0 || absolutePath == NULL) {
+         respondWithJsonError(response, "Failed to idenity the folder pointed", 400, "Bad Request");
```

Comment:
Don't you need to return from here?

Reply:
hmm it is a void function ?

Reply:
I understand that. If you continue execution after you finished the response and then try finishing it again, bad things will happen.

Reply:
You can still return from a void function. You can't return anything, but you can exit the function.

Reply:
sure @ifakhrutdinov